index.ts

```
@@ -0,0 +1,134 @@
+ } catch (error) {
+  console.error(error);
+  res.status(500).json({ message: 'Error fetching posts' });
+ }
+});
+
+// Route to get a single published post
+app.get('/posts/:id', async (req: Request, res: Response) => {
+ const { id } = req.params;
+
+ const parsedId = parseInt(id, 10);
+ if (isNaN(parsedId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)